spec/mongo/collection_spec.rb

```diff
@@ -2000,6 +2000,47 @@ def generate
   end
 
   describe '#count_documents' do
+
+        context 'no argument provided' do
+          let(:view) do
+            described_class.new(authorized_collection, {}, {})
+          end
+
+          before do
+            authorized_collection.delete_many
+          end
+
+          context 'empty collection' do
+            it 'returns true' do
+              expect(view.count_documents()).to eq(view.count_documents({}))
```

Comment:
I don't think you need `view` here, since you're just testing the `count_documents` method on the collection (you used it correctly in the other two tests). You can also remove the `view` helper (the `let(:view)` block).